web/modules/custom/usagov_login/src/Form/LoginSettingsForm.php

```diff
@@ -37,7 +36,7 @@ public function __construct(ConfigFactoryInterface $config_factory, StateInterfa
    * {@inheritdoc}
    */
   #[\Override]
-  public static function create(ContainerInterface $container) {
+  public static function create(ContainerInterface $container): LoginSettingsForm|ConfigFormBase|static {
```

Comment:
`static` is sufficient for phpstan to figure out the return type, since the method instantiates the object via `return new static(`